include/lemlib/timer.hpp

```diff
@@ -10,25 +11,25 @@ class Timer {
          *
          * @param time how long to wait, in milliseconds
          */
-        Timer(uint32_t time);
+        Timer(Time time);
         /**
          * @brief Get the amount of time the timer was set to
          *
-         * @return uint32_t time, in milliseconds
+         * @return Time time, in milliseconds
```

Comment:
```suggestion
         * @return Time time
```
get rid of unit in description